package.json

```diff
@@ -32,13 +32,12 @@
   "dependencies": {
     "@octokit/graphql-schema": "^15.25.0",
     "@octokit/rest": "^21.0.2",
-    "@sinclair/typebox": "0.34.3",
-    "@ubiquity-os/plugin-sdk": "^1.1.0",
+    "@sinclair/typebox": "0.34.11",
```

Comment:
I think we should leave it at `0.34.3` same as in SDK because I noticed even between patch releases there are Typescript errors when passing the schema to `createPlugin`

Reply:
Reverted the version.

Reply:
Ideally I'd remove the package from plugin's dependencies and somehow use SDK's version, that way we don't need to change all plugins when we change the version in the SDK

Reply:
Maybe we could list it in the `peerDepencies` so the same version shall be required in the plugin when installing the sdk?